c/zss.c

```diff
@@ -405,7 +405,7 @@ static hashtable *getServerUserTimeoutsHt(ShortLivedHeap *slh, const char *filen
     if (jsonIsObject(serverTimeouts)) {
       serverTimeoutsJsonObject = jsonAsObject(serverTimeouts);
     } else {
-      zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_SEVERE, ZSS_LOG_FILE_EXPECTED_TOP_MSG, filename);
+      zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_WARNING, ZSS_LOG_FILE_EXPECTED_TOP_MSG, filename);
```

Comment:
At this point, the file exists but isn't formatted correctly or isn't a valid JSON so we do need a warning